src/StatusQ/Controls/StatusAssetSelector.qml

```diff
@@ -18,7 +18,7 @@ Item {
     }
     // Define this in the usage to get balance in currency selected by user
     property var getCurrencyBalanceString: function (currencyBalance) { return "" }
-    implicitWidth: 86
+    implicitWidth: 106
```

Comment:
This should rather imho get its natural width based on the contents (childrenRect.width)